src/main/java/org/pcj/internal/InternalPCJ.java

```diff
@@ -117,10 +120,10 @@ static void start(Class<? extends StartPoint> startPointClass,
 
             /* Starting execution */
             if (isCurrentJvmNode0) {
-                LOGGER.log(Level.INFO, "Starting {0}"
+                LOGGER.log(Level.INFO, "Starting with instance from {0}"
                                 + " with {1,number,#} {1,choice,1#thread|1<threads}"
                                 + " (on {2,number,#} {2,choice,1#node|1<nodes})...",
-                        new Object[]{startPointClass.getName(),
+                        new Object[]{startPointFactory,
```

Comment:
One thing I am worried about is that this change may obscure starting point class name in case someone uses lambda or doesn't override `toString`.

If this is a critical information I can propose two solutions:
- Force toString implementation in the factory interface. However in that case it will no longer be a functional interface.
- Create list of starting points, get their type here and pass them to pcjThreads.